librz/bp/bp_plugin.c

```diff
@@ -39,9 +39,8 @@ RZ_API int rz_bp_plugin_add(RzBreakpoint *bp, RzBreakpointPlugin *foo) {
 	return true;
 }
 
-RZ_API int rz_bp_use(RzBreakpoint *bp, const char *name, int bits) {
+RZ_API int rz_bp_use(RzBreakpoint *bp, const char *name) {
```

Comment:
Doxygen, please.

Reply:
also RZ_NONNULL, and asserts if they are not there